com.unity.render-pipelines.high-definition/Documentation~/Decal-Shader.md

```diff
@@ -31,7 +31,7 @@ These properties allow you to set the inputs that affect the behavior of the dec
 | **Metallic**                    | Use the slider to set the strength of the metallic effect of the decal. Choose a value from 0 and 1 where 0 means no effect and 1 means full effect.<br />This property only appears when you enable the **Metal and Ambient Occlusion properties** checkbox in your [HDRP Asset](HDRP-Asset.md#Decals). |
 | **Ambient Occlusion**           | Use the slider to set the strength of the ambient occlusion effect of the decal.<br />This property only appears when you enable the **Metal and Ambient Occlusion properties** checkbox in your [HDRP Asset](HDRP-Asset.md#Decals). |
 | **Smoothness**                  | Use the slider to set the strength of the smoothness of the decal. |
-| **Metallic Scale**              | Use the slider to set the strength of the metallic effect of the decal. This targets the values of the Metallic map in the Red channel of the **Mask Map**. Choose a value from 0 and 1 where 0 means no effect and 1 means full effect.<br />This property only appears when you enable the **Metal and Ambient Occlusion properties** checkbox in your [HDRP Asset](HDRP-Asset.md#Decals). |
+| **Metallic Remapping**          | Remaps the metallic values from the **Mask Map** to the range you specify. Rather than [clamping](https://docs.unity3d.com/ScriptReference/Mathf.Clamp.html) values to the new range, it condenses the original range down to the new range uniformly.<br />This property only appears when you enable the **Metal and Ambient Occlusion properties** checkbox in your [HDRP Asset](HDRP-Asset.html#Decals). |
```

Comment:
Please, also add this in what's new (that's we have range remapping for metal)  and in upgrade guide (that we have added a new property and rename the one in decal)